hdtv/rootext/dlmgr.py

```diff
@@ -87,15 +96,23 @@ def RebuildLibraries(dir, libraries=None):
     for name in libraries or modules:
         BuildLibrary(name, dir)
 
+
 def BuildLibrary(name, dir):
     srcdir = os.path.join(os.path.dirname(__file__), name)
     # with tempfile.TemporaryDirectory() as tmpdir: TOOD: Not available in python2
     tmpdir = tempfile.mkdtemp()
-    subprocess.check_call(['cmake', srcdir, '-DCMAKE_INSTALL_PREFIX=%s' % dir, '-DCMAKE_BUILD_TYPE=Release'],
-                          cwd=tmpdir)
-    subprocess.check_call(['cmake', '--build', '.', '-j'], cwd=tmpdir)
-    subprocess.check_call(['cmake', '--build', '.', '--target', 'install'], cwd=tmpdir)
+    subprocess.check_call(
```

Comment:
Codacy found an issue: [subprocess call - check for execution of untrusted input.](https://app.codacy.com/manual/janmayer/hdtv/pullRequest?prid=6293112)